contracts/scripts/native_solc_compile_all_functions

```diff
@@ -33,7 +33,8 @@ export SOLC_VERSION=$SOLC_VERSION
 
 compileContract v1_X dev/v1_X/libraries/FunctionsRequest.sol
 compileContract v1_X dev/v1_X/FunctionsRouter.sol
-compileContract v1_X dev/v1_X/FunctionsCoordinator.sol
```

Comment:
Do you need to keep the compile command around? You wouldn't change the old files, would you? For CCIP, we just keep the wrapper but not the generation (or even the contract)

Reply:
That's a good point. It shouldn't ever change